### ISSUE TITLE
Set-JiraIssue : A parameter cannot be found that matches parameter name 'Priority'.

### ISSUE THREAD

**Issue by tibereus** [AtlassianPS/JiraPS#466]
$parameters = @{
    Priority = "10202"
    }
   Set-JiraIssue @parameters -Issue $TicketKey 

Returns
Set-JiraIssue : A parameter cannot be found that matches parameter name 'Priority'.

For -Issue   I tried both
    $JiraTicketObj = Get-JiraIssue $TicketList[$x]
    $TicketKey = $JiraTicketObj.Key

**Comment by milesgratz** [comment]
Agreed that it should be in the module, but my workaround for this was:

```
$issue = Get-JiraIssue -Key "XX-123"
$parameters = @{
    Fields = @{
        'priority' = @{
            'id' = '2'
            'name' = 'Critical'
        }
    }
}
$issue | Set-JiraIssue @parameters
```

You'll need to update the 'id' and 'name' values based on your environment: 

```
Get-JiraPriority
```

**Comment by tibereus** [comment]
Very nice, thanks for that.
